### DISCUSSION TITLE
Stack of Toasts

### DISCUSSION THREAD

**Discussion by emanuelnogueiras** [livewire/flux#137] [Feature Requests] [answered]
I think it's not just happening to me, but when more than one toast is triggered at the same time, or one right after the other quickly, only one is displayed, showing the data of the last one sent. It would be great if it were a stack of toasts... so if more than one is sent, they accumulate in a stack.

**Comment by joshhanley** [comment]
Thanks for reporting! We have converted this issue to a feature request for now, so we can focus on fixing any bugs first.

**Reply by lordisp** [reply]
something like this https://devdojo.com/pines/docs/toast

**Reply by kerkness** [reply]
A react based one which works nice as well
https://notistack.com/features/basic

**Reply by colinmackinlay** [reply]
this alpine js one is also nice.....
https://alpinejs.dev/component/notifications

**Comment by rikodot** [comment]
Any update on this? Really useful feature...

**Comment by vitalijalbu** [comment]
native one
https://www.penguinui.com/components/toast-notification

**Reply by helderneves91** [reply]
With a sound! that's awesome :O never saw with a sound...

**Comment by helderneves91** [comment]
This feature should be prioritized. I omit some useful alerts to the user since only the last is shown.

**Comment by helderneves91** [comment]
Up! Just making sure this is addressed. 👍🏻

**Reply by joshhanley** [reply]
@helderneves91 it's now the second top requested feature, so we're definitely aware of it 😁

<img width="904" alt="image" src="https://github.com/user-attachments/assets/4c3d8c6c-0397-424c-be7f-303a3062e08d" />

**Comment by untitledpng** [comment]
+1 would really love this to be added!

**Comment by tom-rassaerts** [comment]
+1 :)

**Comment by lopatin96** [comment]
Any update on this? Really useful feature...

**Comment by bobik1337** [comment]
Yea, really helpful feature would be. Please

**Comment by Uhasith** [comment]
This is needed !

**Comment by helderneves91** [comment]
Will this be shipped on v4?

**Comment by joshhanley** [comment, marked as answer]
We've just released toast stacks in v2.2.5! You can find the documentation here https://fluxui.dev/components/toast#stack

**Reply by jeffchown** [reply]
Nice!  Thanks, @joshhanley 👍

**Reply by colinmackinlay** [reply]
Thanks @joshhanley 🤩

**Reply by Uhasith** [reply]
Thanks bro !!! @joshhanley
